build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/RestrictedBuildApiService.java

```diff
@@ -128,6 +128,7 @@ private static ListMultimap<Class<?>, String> createLegacyRestTestBasePluginUsag
         map.put(LegacyRestTestBasePlugin.class, ":x-pack:plugin:ilm:qa:multi-cluster");
         map.put(LegacyRestTestBasePlugin.class, ":x-pack:plugin:ilm:qa:multi-node");
         map.put(LegacyRestTestBasePlugin.class, ":x-pack:plugin:ilm:qa:rest");
+        map.put(LegacyRestTestBasePlugin.class, ":x-pack:plugin:inference:qa:mixed-cluster");
```

Comment:
I think we don't need this since we are using the new test framework?

Reply:
Correct. This should be unnecessary.